projects/js-packages/publicize-components/src/components/admin-page/toggles/social-module-toggle/index.tsx

```diff
@@ -88,22 +89,27 @@ const SocialModuleToggle: React.FC = () => {
 			onChange={ toggleModule }
 		>
 			<Text className={ styles.text }>
-				{ __(
-					'When enabled, you’ll be able to connect your social media accounts and send a post’s featured image and content to the selected channels with a single click when the post is published.',
-					'jetpack-publicize-components'
-				) }
+				{ ! is_wpcom
+					? __(
+							'When enabled, you’ll be able to connect your social media accounts and send a post’s featured image and content to the selected channels with a single click when the post is published.',
+							'jetpack-publicize-components'
+					  )
+					: __(
+							'Connect your social media accounts and send a post’s featured image and content to the selected channels with a single click when the post is published.',
+							'jetpack-publicize-components'
+					  ) }
 				&nbsp;
 				<ExternalLink href="https://jetpack.com/redirect/?source=social-plugin-publicize-support-admin-page">
```

Comment:
We need to update these on WPCOM, or we might be able to handle it via the Jetpack redirect code. I'm not sure if we have the current site type.